### ISSUE TITLE
Liquibase ignores the string after first closing brace for sql that contains "${string}}"

### ISSUE THREAD

**Issue by gaylenesia** [liquibase/liquibase#3014]
## Environment

**Liquibase Version**: 4.12.0

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: MS SQL 2016

**Operating System Type & Version**: Windows 10

**Infrastructure Type/Provider**: VM

## Description

Liquibase ignores the string after first closing brace

## Steps To Reproduce

Run a simple sql for SQL Server:
DECLARE @test nvarchar(max) = N'${test}}'

## Actual Behavior
It returns error: Unexpected error running Liquibase: Unclosed mark after the character string '${test}'. [Failed SQL: (105) DECLARE @test nvarchar(max) = N'${test}]

## Expected/Desired Behavior
The sql should be executed successfully.

**Comment by FBurguer** [comment]
Looks like this issue got fixed on (at least) `4.14`. 
Running this changeset:
```
    <changeSet author="test" id="1">
        <sql dbms="mssql">
        DECLARE @test nvarchar(max) = N'${test}}'
        </sql>
    </changeSet>
     <changeSet author="test" id="2">
        <output>My ${test} message</output>
    </changeSet>
```
I get the next output 
Running Changeset: changelog.mssql.xml::1::test
Running Changeset: changelog.mssql.xml::2::test
`My ${test} message`

**Comment by gaylenesia** [comment]
Great! Thank you for the update
